### ISSUE TITLE
Except it probably waited for more than ten seconds. But yeah that's the idea.

### ISSUE THREAD

**Issue by SoniEx2** [fasterthanlime/feedback#67]
**Where's the problem at?**
[Please link to the article](https://fasterthanli.me/articles/the-curse-of-strong-typing)

**No, seriously, where is it?**
Copy/paste the paragraph the problem is in, maybe highlight what's wrong?

> Except it probably waited for more than ten seconds. But yeah that's the idea.

**Describe the issue and a fix if you know it**
In plain English, explain why this is wrong, and how to fix it (if you know how).

`Duration::from_secs(10_000)` is not 10 seconds, but 10000 seconds.

**Comment by SoniEx2** [comment]
(we hope this is okay. let us know if you don't want us to do this?)

**Comment by fasterthanlime** [comment]
(I do want you to do this, that's why there's a button there! I'll do a second pass tomorrow and probably catch most silly typos but your reports are very much appreciated)

(Also your previous report was marked closed without any comments but that's just GitHub automation when I push a "close XXX" commit from the private repo that actually contains the articles. It looks weird for you, sorry!)
